### PR TITLE
feat: deploy ETH warp route between `plumetestnet` and `sepolia`

### PR DESCRIPTION
**Pull request by aroralanuk**
### Description

PlumeTestnet <> Sepolia (Synthetic vs Native)

### Drive-by changes

Added artifacts to core deployment for sepolia for adding plume which I missed in the original core deployment PR.

### Related issues

- fixes https://github.com/hyperlane-xyz/issues/issues/1014

### Backward compatibility

Yes

### Testing

Manual

### CODE REVIEW

File: typescript/sdk/src/consts/environments/testnet.json [hunk 1]
```diff
@@ -105,7 +105,9 @@
     "fallbackRoutingHook": "0x17Dc724B7a2F09141C13b8AC33B396073785c2BC",
     "testRecipient": "0xeDc1A3EDf87187085A3ABb7A9a65E1e7aE370C07",
     "testTokenRecipient": "0x031AD9c560D37baC7d6Bd2d27A2443bAfd10101A",
-    "routingIsmFactory": "0x3F100cBBE5FD5466BdB4B3a15Ac226957e7965Ad"
+    "routingIsmFactory": "0x3F100cBBE5FD5466BdB4B3a15Ac226957e7965Ad",
+    "interchainSecurityModule": "0x958124472b14B7940Ed5317C44a2508791dB1d48",
+    "pausableHook": "0xa68022e53Fd28119D07C8336a8eC84A298Fd38Fd"
```
Comment (jmrossy) [inline typescript/sdk/src/consts/environments/testnet.json:110]: Was it intentional to add these? What does ISM represent in this context? We used to have ISM addrs in here but they were removed.

Comment (aroralanuk) [reply typescript/sdk/src/consts/environments/testnet.json:110]: Yes, I've was running forked tests with writing artifacts enabled and it populates inside other chains as well.

**Review by jmrossy — APPROVED** [review]

File: typescript/infra/scripts/deploy.ts [hunk 2]
```diff
@@ -82,22 +82,29 @@ async function main() {
...
     };
     config = {
-      inevm,
-      ethereum,
+      plumetestnet,
+      sepolia,
     };
     deployer = new HypERC20Deployer(multiProvider, ismFactory);
+    console.log('config: ', JSON.stringify(config, null, 2));
+    // return;
```
Comment (jmrossy) [inline typescript/infra/scripts/deploy.ts:107]: TODO: cleanup before merge?

File: typescript/infra/config/environments/testnet4/core/verification.json [hunk 3]
```diff
@@ -3709,6 +3709,18 @@
...
+      "name": "PausableHook",
+      "address": "0xa68022e53Fd28119D07C8336a8eC84A298Fd38Fd",
+      "constructorArguments": "",
+      "isProxy": false
+    },
+    {
+      "name": "PausableHook",
+      "address": "0xa68022e53Fd28119D07C8336a8eC84A298Fd38Fd",
+      "constructorArguments": "",
+      "isProxy": false
```
Comment (paulbalaji) [inline typescript/infra/config/environments/testnet4/core/verification.json:3723]: could you de-duplicate as a drive-by?

File: typescript/infra/config/environments/testnet4/warp/addresses.json
Comment (yorhodes) [inline typescript/infra/config/environments/testnet4/warp/addresses.json:1]: could add this as a CI fork test pretty easily

File: typescript/infra/config/environments/testnet4/warp/verification.json
Comment (yorhodes) [inline typescript/infra/config/environments/testnet4/warp/verification.json:1]: why are there 2 HypERC20s on plume?

**Review by yorhodes — APPROVED** [review]

**Review by tkporter — APPROVED** [review]
lgtm but can you add https://www.npmjs.com/package/json-stable-stringify#space for the agent config generation?

File: typescript/infra/config/environments/testnet4/core/verification.json (hunk 3 above)
Comment (aroralanuk) [reply typescript/infra/config/environments/testnet4/core/verification.json:2359]: I think it's fine to have duplicates in the verification file specifically because I don't assume uniqueness here, just use the last item.

File: typescript/infra/config/environments/testnet4/warp/verification.json
Comment (aroralanuk) [reply typescript/infra/config/environments/testnet4/warp/verification.json:1]: removed

File: typescript/infra/scripts/deploy.ts (hunk 2 above)
Comment (aroralanuk) [reply typescript/infra/scripts/deploy.ts]: cleanup done
